clients/src/test/java/org/apache/kafka/common/requests/RequestResponseTest.java

```diff
@@ -1921,7 +1929,7 @@ private CreateAclsResponse createCreateAclsResponse() {
         return new CreateAclsResponse(new CreateAclsResponseData().setResults(asList(
             new CreateAclsResponseData.AclCreationResult(),
             new CreateAclsResponseData.AclCreationResult()
-                .setErrorCode(Errors.INVALID_REQUEST.code())
+                .setErrorCode(Errors.NONE.code())
```

Comment:
pardon me, why to make this change?

Reply:
I wanted to ensure that for error code fields in all RPCs we were including NONE in the count. I would have factored out a method taking the error code to use if any of the rest of the tests depended on the actual error code used, but they don't. We'd need to revisit this if [KIP-636](https://cwiki.apache.org/confluence/display/KAFKA/KIP-636%3A+Make+RPC+error+codes+and+messages+tagged+fields) were ever accepted (since it would interfere with coverage of the serialization test), but it seems harmless for the time being.